keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -255,12 +255,25 @@ spec:
   secretRef: [10]
     name:
     namespace:
-  bucketNameActual: [11]
-  bucketIdentifier: [12]
-  parameters: [13]
+  azureBlobIdentity: [11]
```

Comment:
should the 3 store providers be under an umbrella stanza, eg. `bucketSource` or `bucketContentIdentity` ...?

Reply:
for PVs, in code they are, but at the API they're embedded.  I figured we'd follow a similar pattern